### DISCUSSION TITLE
Mouse mode for tablet mode

### DISCUSSION THREAD

**Discussion by K9itten** [ppy/osu#22800] [Ideas] [answered]
Hello if possible will be mouse mode added to the tablet mode (tablet cursor works like mouse one [doesnt teleport on other tablet position click]) ?

**Comment by bdach** [comment]
Turn off the tablet handler in the game settings. I think that should have the desired effect.

**Reply by K9itten** [reply]
This will sadly disable th whole input from tablet thank you anyways

**Comment by ItsShamed** [comment, marked as answer]
Choosing output modes such as Relative Mode is not supported yet, and I don't know if there are any plans to do so. For the time being, you can disable the in-game Tablet driver and use the standalone [OpenTabletDriver](https://github.com/OpenTabletDriver/OpenTabletDriver/) which has support for switching output modes.

**Reply by K9itten** [reply]
Thank you very much it helped me so much!
